submissions/BlueLamantine/memory-pair-game/js/index.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+class General {
+  constructor(startBtn) {
+    this.startButton = document.querySelector(startBtn);
+    this.cardAreas = document.querySelectorAll('.block');
+    this.gameTheme = new Audio('./audio/game_theme.mp3');
+  }
+
+  playAudio() {
+    this.gameTheme.loop = true;
+    this.gameTheme.volume = 0.2;
+    this.gameTheme.muted = false;
+    this.gameTheme.play();
+
+    document.querySelector('#musicControl').addEventListener('click', e => {
+      e.preventDefault();
+      if (this.gameTheme.muted) {
+        this.gameTheme.muted = false;
+        e.target.classList.replace('fa-volume-off', 'fa-volume-up');
+      } else {
+        this.gameTheme.muted = true;
+        e.target.classList.replace('fa-volume-up', 'fa-volume-off');
+      }
+    });
+  }
+  setGameGonfig() {
+    this.startButton.addEventListener('click', e => {
+      e.preventDefault();
+      this.startButton.classList.add('hidden');
+      document
+        .querySelector('#modal')
+        .classList.replace('popup-active', 'popup');
+      document
+        .querySelector('#map')
+        .classList.replace('map-preload', 'map-load');
+      this.cardAreas.forEach(block => {
+        block.classList.replace('block-preload', 'block-load');
+      });
+      new Render(this.cardAreas).renderCards();
+      new Game().startGame();
+      this.playAudio();
+    });
+  }
+}
+
+class Render {
+  constructor(areas) {
+    this.cardsArray = [
+      {
+        name: 'pawtter',
+        path: 'img/cats/pawtter.png',
+      },
+      {
+        name: 'hairmeowne',
+        path: 'img/cats/hairmeowne.png',
+      },
+      {
+        name: 'ron',
+        path: 'img/cats/ron.png',
+      },
+      {
+        name: 'lord',
+        path: 'img/cats/lord.png',
+      },
+      {
+        name: 'sevepuss',
+        path: 'img/cats/sevepuss.png',
+      },
+      {
+        name: 'dumpurrdore',
+        path: 'img/cats/dumpurrdore.png',
+      },
+    ];
+    this.areas = areas;
+    this.pawsInBlock = 10;
+    this.cardsInBlock = 3;
+  }
+
+  shuffleCards() {
+    const shuffleArray = this.cardsArray.concat(this.cardsArray).sort(() => {
```

Comment:
You can simplify this expression
```suggestion
    const shuffleArray = this.cardsArray.concat(this.cardsArray).sort(() => 0.5 - Math.random())
```

Reply:
thanks for a tip!